Atomic/diff.py

```diff
@@ -8,6 +8,12 @@
 from . import Atomic
 
 class Diff(Atomic):
+
+    def diff_tty(self):
```

Comment:
I guess @rhatdan  has strong feelings it should be this way.  I'll defer to him.